setup.py

```diff
@@ -30,8 +21,5 @@ def run(self):
     ],
     install_requires=['future'],
```

Comment:
Is `future` still needed to install things?

Reply:
Hi Jack, its not required, i will remove it.

Reply:
Hi Jack,

I have made the following changes: 
- In Travis.yml, removed and added the versions (in travis.yml) that are listed in the link that you provided,
- In Flatten.py, removed "from past.builtins import basestring" and added "from past.builtins import str" as basestring is not compatible with Python 3.
- In Setup.py, removed "from setuptools.command.build_py import build_py"

Thanks,
Vijay